source/agora/node/Runner.d

```diff
@@ -237,9 +250,11 @@ private TLSContext getTLSContext ()
         const cert_path = cert_search_paths[idx] ~ cert_file;
         const key_path = key_search_paths[idx] ~ key_file;
 
+        log.info("Checking for {} and {}", cert_path, key_path);
```

Comment:
I think we shouldn't print all the lookup, it's not quite interesting for the user.

Reply:
Removed